data/presets/natural/rock.json

```diff
@@ -13,7 +20,13 @@
     "terms": [
         "boulder",
         "stone",
-        "rock"
+        "attached rock",
+        "cairn",
+        "bedrock",
+        "outcrop",
+        "exposed rock",
+        "decorative boulder",
+        "landscape boulder"
     ],
-    "name": "Attached Rock / Boulder"
+    "name": "Rock"
```

Comment:
> Just "Rock" for natural=rock. This is the more common tag and people have been using it for a variety of types of rocks so the simple name seems OK as long as the terms are there.

Well this would lead to `natural=rock` being tagged (a lot) on free-standing non-attached rocks which seems to be wrong according to the previous discussion/research.

If I understand your point correctly, you are saying that this is what is happening anyway already a lot, so renaming the preset name to just "Rock" is merely following the de-facto tagging situation and **not** redefining the meaning of a tag. The latter, we absolutely do not want for obvious reasons that I hope do not need explaining.

What I am missing is proof for the claim that this is indeed the de-facto tagging situation and if you have proof, it should first be documented in the wiki. How much percent of `natural=rock` is not really a rock attached to bedrock and how much percent is on the other hand tagged correctly according to the wiki definition? Without this, we cannot simply shrug this distinction away in an iD preset PR because "it has never been tagged correctly anyway". After all, the very first two sentences of the wiki article mention this distinction:

> [natural](https://wiki.openstreetmap.org/wiki/Key:natural)=rock describes a notable rock feature or small group of rocks, attached to the underlying bedrock, mostly used on a single node element.
> 
> On the contrary, single boulders **not attached** to the bedrock are better tagged as [natural](https://wiki.openstreetmap.org/wiki/Key:natural)=[stone](https://wiki.openstreetmap.org/wiki/Tag:natural%3Dstone).

... and iD (presets), too, has been denoting the meaning of `natural=rock` correctly according to the wiki definition since the preset had been added.

Reply:
Good point - I have a strong suspicion the de facto use is for any rock due to how the word is understood in English, but I will do some more formal queries on this for documentation first.